### PR TITLE
Merge Hashing

### PR DESCRIPTION
**Pull request by asciarp08**

### CODE REVIEW

File: Hashing/Fraction_Interviewbit.cpp [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+            res.push_back(')');
+            break;
+        }
+        
+        m[rem] = res.size();
+        rem *= 10;
+        res.push_back('0' + rem/d);
+    }
+    return res;
+}
```
Comment (vanshikaarora) [inline Hashing/Fraction_Interviewbit.cpp]: This code doesn't compiles successfully. The headers are missing.
Same for other classes
@asciarp08 could you please look into this?

Comment (asciarp08) [reply Hashing/Fraction_Interviewbit.cpp]: This is only the function. Need not to write the full code. Paste it in interview bit and it will compile.

Comment (vanshikaarora) [reply Hashing/Fraction_Interviewbit.cpp]: Could you please add a compiles c++ file instead of the function

Comment (asciarp08) [reply Hashing/Fraction_Interviewbit.cpp]: Okay

**Pull request closed without merging**